calculator.html

```diff
@@ -73,14 +72,13 @@
           </li>
         </ul>
       </form>
-      <a href="#" data-bind="click: remove, visible: calculators().length > 1">Remove</a>
+      <a href="#" data-bind="click: removeCalculator, visible: calculators().length > 1">Remove</a>
     </div>
   </div>
   <div class="add-calculator-div">
-
     <button class="add-calculator" data-bind="click: addCalculator">  Add Calculator </button>
   </div>
+  <script type="text/javascript" src="calculator.js"></script>
+  <script type="text/javascript" src="app.js"></script>

```

Comment:
Moved the script tags to within the body. Script tags should appear within head or body.